PUBLISH.md

```diff
@@ -1,15 +1,14 @@
 Steps to publish a new clippy version
 
-1. `cargo test`.
 - Bump `package.version` in `./Cargo.toml` (no need to manually bump `dependencies.clippy_lints.version`).
-- Run `./util/update_lints.py`.
+- Run `./pre_publish.sh`
 - Write a changelog entry.
-- Commit `./Cargo.toml`, `./clippy_lints/Cargo.toml` and `./CHANGELOG.md`.
+- Review and commit all changed files
```

Comment:
I prefer to have one commit just for the version bump itself,that is the `Cargo.toml`s and the changelog.